### DISCUSSION TITLE
Custom trained model

### DISCUSSION THREAD

**Discussion by sailingbo** [KoljaB/RealtimeSTT#36] [Q&A] [answered]
Awesome work on this project...it's exactly what I needed. I was curious if there was a way to use it with a custom trained model. Let's say I have a model on hugging-face I want to use...is it possible?

**Comment by KoljaB** [comment, marked as answer]
Yes, in theory it should work with any CTranslate2-converted Whisper model (never tried that tho). Submit the path to the model directory or a model ID from the HF Hub as model parameter, and you should be good. Some [further info](https://github.com/SYSTRAN/faster-whisper#model-conversion).
